src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/claimantlip/casestate/ClaimantLiPCaseStateManagment.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.civil.handler.callback.user.claimantlip.casestate;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.civil.enums.CaseState;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+
+@Component
+@AllArgsConstructor
+public class ClaimantLiPCaseStateManagment {
+
+    public AboutToStartOrSubmitCallbackResponse caseStateHandlerForClaimantResponse(
+        AboutToStartOrSubmitCallbackResponse response, CaseData caseData) {
+        if ((caseData.isClaimantNotSettlePartAdmitClaim() || caseData.isFullDefence())
```

Comment:
Could you extract the conditions into a private function with a more readable name?

Reply:
To avoid this logic in the handler, I extracted this into service, and a method may be why it is in private and also suggest the name, it already states that 'caseStateHandlerForClaimantResponse' which handles the case state handling for claimant response

Reply:
I was referring to the conditions within the if (explained via slack).

Reply:
With further stories, this handler is going to mess with multiple jobs like logic  for triggering the camunda process, also for case state management .. etc , so to avoid that I created  a separate service  which handles the state case based on the claimant response

Reply:
_caseData.isClaimantNotSettlePartAdmitClaim() || caseData.isFullDefence()_
Unless this could be generalized to a named claim state it could be separated to a method. From clean code perspective, this is OK. Its just 2 booleans after all and method names are clear.

Also is the same condition used somewhere else? If answer is YES - put it to separate function.
If NO - If this IF statement got bigger in the future, THEN put it to separate method. 

at this point context of the condition is clear. Condition is not too long so i would leave it as it is